snap/README.md

```diff
@@ -21,37 +21,70 @@ The snap is published in the snap store at https://snapcraft.io/edgex-app-servic
 You can see the current revisions available for your machine's architecture by running the command:
 
 ```bash
-$ snap info edgex-app-service-configurable
+snap info edgex-app-service-configurable
 ```
 
 The latest stable version of the snap can be installed using:
 
 ```bash
-$ sudo snap install edgex-app-service-configurable
+sudo snap install edgex-app-service-configurable
 ```
 
-The 2.0 (Ireland) release of the snap can be instaing using:
+The 2.1 (Jakarta) release of the snap can be instaing using:
 
 ```bash
-$ sudo snap install edgex-app-service-configurable --channel=2.0
+sudo snap install edgex-app-service-configurable --channel=2.1
```

Comment:
Please replace with the following to make it simpler to maintain, such that we don't have to change this in every release:

A specific version of the snap can be installed by setting the channel, for instance for 2.1 (Jakarta):
```
sudo snap install edgex-app-service-configurable --channel=2.1
```

Reply:
Updated.